config.test.toml

```diff
@@ -0,0 +1,194 @@
+# Supported values are: 'development' and 'production'. Defaults to "development"
+# Env override: GO_ENV
+GoEnv = "development"
+
+# GoGetWorkers specifies how many times you can concurrently
+# go mod download, this is so that low performance instances
+# can manage go get more sanely and not run out of disk or memory.
+# Env override: ATHENS_GOGET_WORKERS
+GoGetWorkers = 30
+
+# ProtocolWorkers specifies how many concurrent
+# requests can you handle at a time for all
+# download protocol paths. This is different from
+# GoGetWorkers in that you can potentially serve
+# 30 requests to the Download Protocol but only 5
+# at a time can stash a module from Upstream to Storage.
+# Env override: ATHENS_PROTOCOL_WORKERS
+ProtocolWorkers = 30
+
+# LogLevel returns the system's exposure to internal logs. Defaults to debug.
+# Supports all logrus log levels (https://github.com/Sirupsen/logrus#level-logging)
+# Env override: ATHENS_LOG_LEVEL
+LogLevel = "debug"
+
+# BuffaloLogLevel returns the log level for logs
+# emitted by Buffalo itself. This is different from our own
+# LogLevel in this file because you might want info level
+# for our codebase, but panic level for buffalo.
+# Env override: BUFFALO_LOG_LEVEL
+BuffaloLogLevel = "debug"
+
+# CloudRuntime is the Cloud Provider on which the Proxy/Registry is running.
+# Currently available options are "GCP", or "none". Defaults to "none"
+# Env override: ATHENS_CLOUD_RUNTIME
+CloudRuntime = "none"
+
+# MaxConcurrency sets maximum level of concurrency
+# Defaults to number of cores if not specified.
+# Env override: ATHENS_MAX_CONCURRENCY
+MaxConcurrency = 4
+
+# The maximum number of failures for jobs submitted to buffalo workers
+# Defaults to 5.
+# Env override: ATHENS_MAX_WORKER_FAILS
+MaxWorkerFails = 5
+
+# The filename for the include exclude filter. Defaults to 'filter.conf'
+# Env override: ATHENS_FILTER_FILE
+FilterFile = "filter.conf"
+
+# Timeout is the timeout for external network calls in seconds 
+# This value is used as the default for storage backends if they don't specify timeouts
+# Defaults to 300
+# Env override: ATHENS_TIMEOUT
+Timeout = 1
+
+# EnableCSRFProtection determines whether to enable CSRF protection.
+# Defaults to false
+# Env override: ATHENS_ENABLE_CSRF_PROTECTION
+EnableCSRFProtection = false
+
+
+[Proxy]
+    # StorageType sets the type of storage backend the proxy will use.
+    # Possible values are memory, disk, mongo, postgres, sqlite, cockroach, mysql
+    # Defaults to mongo
+    # Env override: ATHENS_STORAGE_TYPE
+    StorageType = "memory"
+    # Port sets the port the proxy listens on
+    # Env override: PORT
+    Port = ":3000"
+    # The endpoint for Olympus in case of a proxy cache miss
+    # Env override: OLYMPUS_GLOBAL_ENDPOINT
+    OlympusGlobalEndpoint = "http://localhost:3001"
+    # Redis queue for buffalo workers
+    # Defaults to ":6379"
+    # Env override: ATHENS_REDIS_QUEUE_PORT
+    RedisQueueAddress = ":6379"
```

Comment:
Could you rename to RedisQueuePort?

Reply:
I believe we can provide it with any valid address, I don't think it needs to be local.